assets/scripts/index.js

```diff
@@ -0,0 +1,126 @@
+$(document).ready(function(){
```

Comment:
@edwartexe regarding code readability, we put one space before opening a bracket and another between the `function` keyword and the opening parenthesis. This also applies in control statements (if, while etc.)
```suggestion
$(document).ready(function () {
```